src/views/JoinForm/JoinForm.js

```diff
@@ -1,10 +1,183 @@
-import React from 'react';
-import { useParams } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import {
+	getFormData,
+	storeFormResponse
+} from 'utils/form-data/form-data';
+import Loading from 'views/Loading/Loading';
+import CurrentQuestion from 'views/JoinForm/CurrentQuestion';
+import {
+	PATH_DASHBOARD,
+	PATH_HOME
```

Comment:
export `Path` object from your helper and use it like `Path.Dashboard` and `Path.Home`, this is easy and more readable.